test/tools/unified-spec-runner/match.ts

```diff
@@ -489,28 +499,34 @@ export function matchesEvents(
   }
 }
 
+function isMongoCryptError(err): boolean {
+  if (err.constructor.name === 'MongoCryptError') {
+    return true;
+  }
+  return err.stack.includes('at ClientEncryption');
```

Comment:
We haven't standardized error handling in the fle module, some of the errors come out as MongoCryptError, but other validation errors are thrown as TypeError. It would be good to standardize these - if we want to preserve the MongoCrypt aspect of the error that's fine, but we could consider rewrapping the key management api errors in the driver.

@baileympearson - tagging you specifically since you did most of the key management work